CHANGELOG.rst

```diff
@@ -15,48 +15,146 @@ The following notes provide some details on what has been revised for each
 version in reverse chronological order (most recent version at the top
 of the list).
 
-3.2.2 (unreleased)
+3.3.0 (28-Sep-2021)
 ==================
+This version includes all the functionality needed to generate
+source catalogs, both point source and extended (segment) source
+catalogs, during single-visit mosaic (SVM) processing.  In fact,
+
+- Updated code to work with Python >= 3.7
+- GAIAeDR3 catalog now the initial catalog of choice for a posteriori alignment
+  during standard pipeline processing, as well as for SVM/MVM processing.
+- SVM/MVM processing will loop over catalogs, fit methods and fit geometries in
+  looking for successful fit, using first successful fit it computes.
```

Comment:
looking for **a** successful, using **the** first 

Since the description is a real sentence, one may as well use the "articles".

Reply:
Corrected in c11168d